### PR TITLE
Sylvester's sequence

### PR DESCRIPTION
**Pull request by EgbertRijke**
This PR adds Sylvester's sequence to agda-unimath.

### CODE REVIEW

File: src/elementary-number-theory/sylvesters-sequence.lagda.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+numbers `s i` for `i < n`, i.e.,
+
+$$
+  s_n := 1+\left(\prod_{i<n}s_i\right).
+$$
+
+The first few entries in this sequence are `s 0 = 2`, `s 1 = 3`, `s 2 = 7`, and
+`s 3 = 43`.
+
+Sylvester's sequence is listed as A000058 in the [OEIS](literature.oeis.md)
```
Comment (fredrik-bakke) [inline src/elementary-number-theory/sylvesters-sequence.lagda.md:35]:
```suggestion
Sylvester's sequence is listed as [A000058](https://oeis.org/A000058) in the [OEIS](literature.oeis.md)
```

**Review by fredrik-bakke — APPROVED** [review]

File: src/elementary-number-theory/sylvesters-sequence.lagda.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+open import elementary-number-theory.products-of-natural-numbers
+
+open import univalent-combinatorics.standard-finite-types
+```
+
+</details>
+
+## Idea
+
+{{#concept "Sylvester's sequence" Agda=sylvesters-sequence-ℕ}} is the
```
[the remark this answers is not in the document]
Comment (EgbertRijke) [reply src/elementary-number-theory/sylvesters-sequence.lagda.md:21]: is WDD a typo or is it an entry that I don't know about?

[the remark this answers is not in the document]
Comment (fredrik-bakke) [reply src/elementary-number-theory/sylvesters-sequence.lagda.md:21]: the first one

[the remark this answers is not in the document]
Comment (fredrik-bakke) [reply src/elementary-number-theory/sylvesters-sequence.lagda.md]: I misremembered the field name, sorry

[the remark this answers is not in the document]
Comment (EgbertRijke) [reply src/elementary-number-theory/sylvesters-sequence.lagda.md]: all good

File: src/foundation/sequences.lagda.md [hunk 3]
```diff
@@ -17,7 +17,14 @@ open import foundation-core.function-types
 
 ## Idea
 
-A **sequence** of elements in a type `A` is a map `ℕ → A`.
+A {{#concept "sequence" Agda=sequence}} of elements of type `A` is a map `ℕ → A`
+from the [natural numbers](elementary-number-theory.natural-numbers.md) into
+`A`.
+
+For a list of sequences from the
```
Comment (fredrik-bakke) [inline src/foundation/sequences.lagda.md:24]: to disambiguate from the main concept of the file:

```suggestion
For a list of number sequences from the
```

**Comment by EgbertRijke** [comment]
Thanks for the review!

**Pull request merged**